include/IECore/CurvesAlgo.h

```diff
@@ -44,7 +44,7 @@
 namespace IECore
 {
 
-namespace CurveAlgo
+namespace CurvesAlgo
```

Comment:
Ah, perfecto. Can we rebase this in though? Shouldn't be too hard, just ensure you have separate commits for each file touched here, and then they can all slot into previous commits easily.